README.md

```
@@ -71,4 +88,22 @@ run-fit
 
 ## 기여도와 역할
 
+|                                                                  팀원                                                                   | 역할                                                                                                                                                                                                                      |
+| :-------------------------------------------------------------------------------------------------------------------------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|    <img src="https://avatars.githubusercontent.com/u/52899088?v=4" width="80" height="80"> <br> **[석준](https://github.com/KSJ27)**    | - 테스트 환경 설정 <br> - 지도·주소 보기·설정, 세션 상세 보기, 세션 생성·수정, 세션 찜하기 구현 <br> - Header, Button, UserAvatar, Tabs, Textarea, Modal 공통 컴포넌트 개발                                               |
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)